test/verify/test_quantizelinear.cpp

```diff
@@ -0,0 +1,24 @@
+
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_quantizelinear : verify_program<test_quantizelinear>
+{
+    migraphx::program create_program() const
+    {
+        migraphx::program p;
+        auto* mm = p.get_main_module();
+
+        migraphx::shape sx{migraphx::shape::float_type, {2, 2, 2}};
```

Comment:
Maybe we can add a test with the sx of type int32 or double, in this way, we can test the apply_quantizelinear() with x and y_scale of different data type.